 \documentclass[9pt,twocolumn,twoside,lineno]{pnas-new}
\templatetype{pnasresearcharticle}

%packages
\usepackage{siunitx}
\sisetup{per-mode = fraction}
\usepackage{leftindex} 
\usepackage{subcaption}
\usepackage{nameref}


\newcommand{\siref}[1]{\textcolor{blue}{S#1}}
\newcommand{\sifig}[1]{\textcolor{blue}{Figure~S#1}}
\newcommand{\sitab}[1]{\textcolor{blue}{Table~S#1}}
\newcommand{\sieq}[1]{\textcolor{blue}{Equation~S#1}}
\newcommand{\sisec}[1]{\textcolor{blue}{#1}}

\newcommand{\panelref}[2]{\autoref{#1}\textcolor{blue}{#2}}

\renewcommand{\thesubfigure}{\Alph{subfigure}}

\acknow{
    We thank Adrian Egli for generously providing the ESBL samples containing the plasmids and Fabienne Benz and Jana Huisman for their assistance with selecting them.
    We also thank Lukas Graz from the Seminar for Statistics (ETHZ) for his statistical consultation. 
    We used Grammarly and OpenAI's ChatGPT for proofreading and grammar checking.
    We thank ETH Zurich for providing funding.}


%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
\matmethods{
    \paragraph{Drugs and Media.}
    In all experiments, we used LB (Sigma L3022) with 25 $\mu g/ml$ (\textit{prevention}~scenario) or 5 $\mu g/ml$ (\textit{containment} and \textit{max-emergence}~scenario) chloramphenicol as a liquid medium and the same LB and drugs with 1.5\% agar as a solid medium.
    Chloramphenicol was added to prevent external contaminations. We could not measure any significant growth effects of the chloramphenicol concentrations on the chloramphenicol-resistant strains (see \sitab{5}).
    We used 80 $\mu g/ml$ ceftazidime as drug A and 40 $\mu g/ml$ tetracycline as drug B, with identical concentrations for liquid and solid media.
    
    \paragraph{Strains and Plasmids.}
    We used two compatible plasmids $p_A$ and $p_B$ derived from samples ESBL9 and ESBL25 from a clinical transmission study \cite{Sutter2016}. 
    Samples were kindly provided by Adrian Egli and sequenced and analysed by Huisman et al. \cite{Huisman2022}. 
    Plasmids $p_A$ and $p_B$ provide (among other resistances) resistance against drug A and drug B, respectively.
    We used these plasmids and the chloramphenicol-resistant host MDS42-YFP \cite{Feher2012} (sensitive to drugs A and B) to create three additional strains by conjugation (\sitab{2}) (i) A-resistant, containing $p_A$; (ii) B-resistant, containing $p_B$; and (iii) AB-resistant, containing both plasmids (see \sisec{SI Methods}).
    
    \paragraph{Treatment arms.}
    We simulated the epidemiological dynamics of six hospital wards \textit{in vitro}, with each ward exploring a different treatment arm:
    (i) control with no treatment, (ii) monotherapy with ceftazidime (mono A), (iii) monotherapy with tetracycline (mono B), (iv) cycling therapy (A, A, B, B, ...), (v) mixing therapy (treatment A and B are randomly assigned daily, without knowledge of prior treatment), and (vi) combination therapy (treating all patients with both drugs, each at full concentration).
    
    \paragraph{Assay plates.}
    Each hospital ward was simulated \textit{in vitro} on a 384-well microtiter plate (Greiner 781186).
    Wells are interpreted as beds in four replicate hospital wards with 94 beds each. 
    The remaining wells contained only growth medium and remained untouched, acting as sentinels for contamination.
    Across all experiments and treatment arms, 2752 control wells were used, 67 of which became contaminated.
    Wells with growth medium but no bacteria represent uninfected patients, whereas wells with growth medium and (resistant or sensitive) bacteria represent infected patients. 
    
    \paragraph{Experimental procedure. \label{par:setup}}
    Experiments were performed using a Tecan Evo 200 automated liquid handling system (Tecan) with an integrated, automated incubator (Liconic STX100, Liconic), a Tecan Infinite F200 spectrophotometer (Tecan), and a camera (Pickolo, SciRobotics).
    
    Every day new assay plates were filled with \SI{45}{\micro \l} fresh medium and \SI{5}{\micro \l} antibiotic stock, according to its designated treatment strategy (see \sifig{1}).
    At each of these transfers, we simulate patients staying overnight in the hospital (passage), the admission and discharge of patients (turnover), and infections between patients (infection).
    Passage, turnover and infections were all done by inoculating the new plate using a pintool with retractable pins, as detailed below, carrying \SI{\approx 0.3}{\micro \litre} drops between wells ($\approx1:150$ dilution) leading to an approximately 6-8 hours exponential phase. 
    The \textit{assay plates} were then incubated at \SI{37}{\celsius} and 95\% relative humidity.
    The incubation duration varied due to variations in the time it takes to set up a new transfer and occasional transfer repetitions made necessary because of machine errors or user mistakes.
    The mean incubation duration was 27 hours.
    
    We initiated the experiment by inoculating one 384-well plate from fresh overnight cultures representing patients from an outside community. 
    We assume that this community is sufficiently large to be unaffected by interactions with the hospital ward. 
    Incoming patients are either uninfected or carry one of the four strains (sensitive, each single resistant or double resistant) and are sampled according to predefined sampling proportions, defining the probability of a resistance profile being admitted to the hospital. (\autoref{tab:exp_par}).
    This initial plate remained untreated and was used as the initial population for all six treatment arms.
    
    
    \paragraph{Turnover.}
    Every transfer, each patient has a turnover probability $\tau$ to be discharged from the hospital and replaced by a newly admitted patient.
    Wells representing staying patients were passed from the previous to the new {\it assay plate} using the pintool.
    Here, the pins for discharged patients are retracted.
    Vacant beds on the {\it assay plate} were then reoccupied by patients from the community analogous to the initial setup.
    
    \paragraph{Infections.}
    To simulate infections, each well has an infection probability $\beta$ to infect another randomly chosen well on the next \textit{assay plate} during the transfer.
    Therefore, each patient can infect at most one other patient per transfer, but several patients could potentially infect the same patient.
    
    \paragraph{Resistance Profiles.\label{par:phenotyping}}
    To assess the resistance profile of each well, we spotted the previous \textit{assay plate} onto four \textit{agar plates}, using the pintool.
    Three plates were treated with antibiotics (A, B, or AB), while one was untreated (none).
    After incubation at \SI{37}{\celsius} and 95\% relative humidity, images of the \textit{agar plates} are taken and analysed using the Pickolo package (SciRobotics, Kfar Saba, Israel). 
    The software automatically detects the presence of colonies at each well position, which we also manually verified. 
    The growth pattern on the four \textit{agar plates} allowed us to determine the resistance profile of a well, which reflects how the well would behave if treated.
    
    By default, we distinguish six resistance profiles (\sitab{6}). 
    The wells may either be 1) uninfected ($U$), 2) exclusively infected with sensitive bacteria ($S$),  3) infected with A-resistant bacteria ($A_r$), 4) infected with B-resistant bacteria ($B_r$), 5) infected with AB-resistant bacteria $(AB_r)$, 6) or be infected with a mixed population containing A-resistant and B-resistant bacteria, but no AB-resistant bacteria $((A\&B)_r)$. 
    The way we classify the resistance profiles of the bacterial population in a well leads to the dominance of resistance, in the sense that a predominantly sensitive population harbouring a resistant minority would be classified as resistant (see \sitab{7}). 
    Any observed growth pattern not corresponding to the six resistance profiles mentioned above is classified as \textit{'other'}. 
    The resistance profile \textit{'other'} primarily occurs when bacterial densities are low (see also \sisec{SI Methods}). 
    
    
    \paragraph{Scenarios.}
    We conducted experiments for three scenarios (\textit{prevention}, \textit{containment}, and \textit{maximum-emergence}) with 14 to 27 transfers each.
    Each experiment was defined by a different parameter set consisting of (i) the infection probability $\beta$ within the hospital, (ii) the turnover probability $\tau$ and (iii) the sampling proportions $c_\phi$ of patients with resistance profile $\phi \in \{U, S, A_r, B_r, AB_r\}$ (see \autoref{tab:exp_par}).
    
    The \textit{prevention}~scenario (\sifig{2}) addresses how the treatment strategies perform with a moderately resistant community and a moderate infection regime in the hospital ward and how well they are able to prevent the upcoming double resistance. 
    
    The \textit{containment}~scenario (\sifig{4}) corresponds to a scenario in which some patients entering the hospital are infected with double-resistant bacteria to compare the ability of treatment strategies to contain the spread of pre-existing double resistance.
    
    During the \textit{maximum-emergence}~scenario (\autoref{fig:exp3}) 50 \% of the incoming patients are infected with A-resistant bacteria, and the other 50 \% are infected with B-resistant bacteria. These conditions maximally favour opportunities for horizontal gene transfer.
    The basic reproduction number was set to $R_0 = 0.5$ (\sieq{1}) to ensure that double-resistant strains are flushed out, reducing the stochastic dependency on earlier emergence events while maintaining a high potential for new emergence.
    
    \paragraph{Instruction Sets.}
    Based on the parameter defined for each experiment (see \autoref{tab:exp_par}), we generated instructions that were passed to the liquid handling platform.
    These instructions specify which patients are passaged or discharged and admitted, who infects whom, and the treatment for mixing therapy. 
    Instructions are randomly generated prior to each transfer. 
    We call the entirety of all instructions that come up during an experimental run an instruction set.
    Instruction sets are identical across all treatment arms and replicates.
    
    \paragraph{Computational Model.}
    We created a stochastic model (\sisec{SI Computational Model}) incorporating 94 \textit{in silico} patients, each capable of adopting one of six resistance profiles $\phi \in \{U, S, A_r, B_r, AB_r, (A_r\&B_r)\}$. 
    The model is structured analogue to the {\it in vitro} experiments (\sifig{1}) and alternates between modelling the transactions between wells and the effect of treatment during incubation. 
    
    Admission and discharge (turnover) were simulated by replacing the resistance profile of the current patient with that of the incoming patient, as defined by the instruction set.
    Infections are simulated by combining the resistance profiles of the receiving well $i$ and the infecting well $j$. 
    The resulting resistance profile $\phi_i + \phi_j$ is determined using the rules based on the dominance of resistance specified in \sitab{9}. 
    Calculations involving more than two resistance profiles apply the associative law and are determined pairwise, e.g. $(U + S) + A_r = S + A_r = A_r$. 
    
    To model treatment effects, we use transition probabilities to assign the post-incubation resistance profile $\phi(\hat{T})$ stochastically based on the treatment and the pre-incubation resistance profile $\phi(T)$. 
    The transition probabilities (\sitab{18} -- \siref{25}) were estimated based on experimental data across all experiments.   
    
    
    \paragraph{\textit{In Silico} Sensitivity Analysis.}
    To augment the experimental data, we conducted an \textit{in silico} sensitivity analysis.
    We randomly generated 10,000 parameter sets with and 10,000 without pre-existing double resistance.
    Turnover and infection probabilities were uniformly sampled $[0.05, 0.95]$, allowing for  $R_0 \in [0.0526, 19]$. 
    The sampling proportions $c_\phi$ for all incoming resistance profiles ($\phi \in \{U, S, A_r, B_r, AB_r\}$) were randomised by sampling a number $n_\phi \in [0, 1]$ from a uniform distribution and subsequently normalising by the sum: $c_\phi = {n_\phi}/{\sum_j n_j}$. 
    We created ten randomised instruction sets for each parameter set and conducted one simulation per instruction set (\sifig{3D}) for 28 transfers. 
    
    For this analysis, the frequency of non-infected individuals during the last four transfers was used as a performance metric for treatment strategies, as it also indirectly reflects the frequency of both double- and single-resistant patients.
    We conducted an ANOVA test to assess if the effect of the treatment strategies significantly ($p<0.05$) influences the frequency of uninfecteds. 
    For significant tests, we proceeded with Tukey's post hoc analysis \((p < 0.05)\), identifying significantly distinct pairs of strategies. 
    Strategies not significantly inferior to others were classified as `winners', while strategies not significantly superior to any were classified as `losers'.
    Strategies that win or lose a parameter set alone are `single winners' or `single losers'. 
    
    \paragraph{Data Availability}
    Experimental data and analysis scripts, as well as code for the computational model, have been deposited in Zenodo (\url{https://doi.org/10.5281/zenodo.14137410}).}

\title{The Impact of Treatment Strategies on the Epidemiological Dynamics of Plasmid-Conferred Antibiotic Resistance}

\author[a,1]{Malte Muetter}
\author[a]{Daniel C. Angst}
\author[a]{Roland R. Regoes}
\author[a]{Sebastian Bonhoeffer}
\affil[a]{Institute of Integrative Biology, Department for Environmental System Science, ETH Zurich, 8092 Zurich, Switzerland}

\leadauthor{Malte Muetter}


% Please include corresponding author, author contribution and author declaration information
\authorcontributions{MM, DCA, RR, and SB designed research; MM and DCA performed experiments; DCA and MM created the used strains; MM developed the mathematical model; MM analyzed data; MM, DCA, RR and SB wrote the manuscript.}
\authordeclaration{The authors declare no competing interest.}
\correspondingauthor{\textsuperscript{1}To whom correspondence should be addressed. E-mail: malte.muetter@env.ethz.ch}

% At least three keywords are required at submission. Please provide three to five keywords, separated by the pipe symbol.
\keywords{Combination Therapy $|$ Antibiotic Resistance $|$ Plasmids $|$ Experimental Epidemiology $|$ Treatment strategies}

\dates{This manuscript was compiled on \today}
\doi{\url{www.pnas.org/cgi/doi/10.1073/pnas.XXXXXXXXXX}}

 \begin{abstract}
        The issue of antibiotic resistance is a critical concern for public health, prompting numerous investigations into the impact of treatment strategies on preventing or slowing down the emergence of resistance. While existing studies have predominantly focused on chromosomal resistance mutations, the consequences of often clinically more relevant plasmid-conferred resistance remain insufficiently explored.
        To address this gap, we conducted three extensive {\it in vitro} experiments utilising a liquid-handling platform. These experiments evaluated the efficacy of five distinct treatment strategies using two antibiotics (tetracycline and ceftazidime) along with two horizontally transmissible clinical resistance plasmids conferring the respective resistances. 
        Among the experimentally investigated treatment strategies, combination therapy proved to be the most effective in preventing the emergence of double resistance while minimising the number of infections.
        To verify the reliability of these findings, we constructed a computational model of our experiments that we parameterised using the experimental data. 
        We employed this model to augment the experimental data by conducting an {\it in silico} parameter sensitivity analysis. 
        The sensitivity analysis corroborated our experimental results, demonstrating that combination therapy consistently outperformed other treatment strategies across a range of parameter values.
    \end{abstract}

\significancestatement{
        Given the rising levels of antibiotic resistance, we need a better understanding of how to use antibiotics sustainably. While most current work on treatment strategies to reduce resistance focuses on chromosomal resistance, we focus on plasmid-borne resistance, which can be transmitted between bacteria. We combine large-scale \textit{in vitro} experiments with a computational model to assess the effectiveness of various treatment strategies in managing preexisting antibiotic resistance and preventing the emergence of new double resistance through horizontal gene transfer. We observed that combination therapy is most effective in suppressing the emergence of plasmid-borne double resistance. This is achieved by decreasing co-colonization with single-resistant strains and reducing the probability of co-colonized patients developing double resistance.
    }

\begin{document}
    \maketitle
    
    \thispagestyle{firststyle}
    \ifthenelse{\boolean{shortarticle}}{\ifthenelse{\boolean{singlecolumn}}{\abscontentformatted}{\abscontent}}{}
    
    \firstpage[6]{4}
     % Use \firstpage to indicate which paragraph and line will start the second page and subsequent formatting. In this example, there are a total of 11 paragraphs on the first page, counting the first level heading as a paragraph. The value {12} represents the number of the paragraph starting the second page. If a paragraph runs over onto the second page, include a bracket with the paragraph line number starting the second page, followed by the paragraph number in curly brackets, e.g. "\firstpage[4]{11}".

    % If your first paragraph (i.e. with the \dropcap) contains a list environment (quote, quotation, theorem, definition, enumerate, itemize...), the line after the list may have some extra indentation. If this is the case, add \parshape=0 to the end of the list environment.
    
    \dropcap{I}n light of the growing threat of antimicrobial resistance (AMR) to human health, various multidrug strategies are being considered to improve the sustainability of antibiotic use.
    These approaches include combination therapy (simultaneous use of multiple antibiotics), mixing therapy (randomly assigning patients to receive different antibiotics), and cycling therapy (alternating between multiple antibiotics over time).

    Combination, originally proposed alongside cycling therapy to prevent biocide resistance in plant pathogens \cite{Kable1980, Delp1980, Skylakakis1981}, was later adopted in human medicine.
    Combination therapy proved its effectiveness in preventing resistance evolution in highly adaptable pathogens such as HIV, \textit{Mycobacterium tuberculosis}, and \textit{Plasmodium falciparum} \cite{Goldberg2012}. 
    However, a recent meta-analysis investigating the effect of combination therapy on resistance across various bacterial infections and antibiotic combinations found no evidence for a difference in the risk of resistance acquisition \cite{siedentop_metaanalysis_2024}. 
    Also, a comprehensive cluster-randomised crossover study comparing mixing and cycling by van Duijn et al. \cite{VanDuijn2018}, spanning nearly two years across eight ICUs, found no significant differences in outcomes.
    
    A review of the available model literature by Uecker et al. \cite{Uecker2021} reveals the complexity and context-dependent efficacy of treatment strategies such as combination, cycling or mixing strategies.
    Yet, theoretical models often identify combination therapy as the best strategy to prevent new resistance \cite{Bonhoeffer1997, Tepekule2017}.
    It remains unclear whether the inconclusive results regarding the effectiveness of multidrug treatment strategies in the literature are due to the theoretical models failing to account for key processes, or if clinical studies lack statistical power, as suggested by Siedentop et al. \cite{siedentop_metaanalysis_2024}.
    This lack of power may be caused by patient and bacterial strain heterogeneity, stochasticity in infection dynamics, and other unknown factors that make it difficult to isolate single effects.
    
    We recently started experiments to make a foray into the large gap between theoretical models and clinical trials.
    In an {\it in vitro} experiment mimicking the epidemiological scenario of transmission in a hospital ward, Angst et al. \cite{Angst2021} investigated the effect of cycling, mixing, and combination therapy on resistance evolution and showed that for chromosomal resistance mutations combination therapy outperformed the other strategies.
    One potential reason why combination therapy succeeded in that study and tends to be superior in mathematical models is that it increases the genetic barrier to resistance by requiring the acquisition of multiple mutations in the same background.
    
    Here, we explore the effect of horizontal gene transfer (HGT) on resistance evolution under treatment by conducting three large-scale \textit{in vitro} experiments. 
    The experiments mimic epidemiological transmission dynamics of symptomatic infections by a focal strain in an intensive care unit (ICU) and include patient discharge and admission, infection between patients, and treatment.
    We use two antibiotics, ceftazidime (A) and tetracycline (B), along with two clinical resistance plasmids \cite{Huisman2022} we call $p_A$ and $p_B$, conferring resistance to the corresponding antibiotics.
    The plasmids are compatible, can conjugate, and were isolated from clinical samples collected and characterised in a study at University Hospital Basel \cite{Sutter2016}.
    We model patients as wells in a 384-well microtiter plate filled with LB medium. 
    These ``patients'' can be infected with a mixture of bacteria, which may carry up to two resistance plasmids. 
    Depending on the presence of bacteria and resistance, we assign each "patient" a resistance profile: uninfected ($U$), sensitive infected ($S$), single-resistant ($A_r$, $B_r$, or $(A_r\&B_r)$), or double-resistant ($AB_r$).
    
    In each experiment, we model six hospital wards to assess the ability of five treatment strategies (mixing, cycling, combination therapy with two antibiotics and two monotherapies with each antibiotic alone) and one control (no antibiotics) to contain the spread of plasmid-borne resistance and prevent the emergence of double resistance. 
    All patients in each ward are treated daily according to the assigned strategy.
    A schematic of the experimental setup is shown in \sifig{1}.
    Each of the three experiments addresses a different scenario (\autoref{tab:exp_par}), varying in patient turnover probability (admission/discharge),  infection probability, and the distribution of resistance profiles for incoming patients (sampling proportions). 
    The \textit{prevention}~scenario addresses a situation with low levels of pre-existing single and no double resistance brought into the hospital ward from the community.
    The \textit{containment}~scenario focuses on the ability of treatment strategies to contain pre-existing double resistance and in the \textit{maximum-emergence}~scenario, we maximised the opportunities for emerging double resistance through HGT by admitting single-resistant patients only.
    
    Alongside our experiments, we created a computational model that mimics the experiment and is parameterised but not fitted using the experimental data.
    We used the model to assess the robustness of our findings to the randomisation of the experimental decisions and conducted an \textit{in silico} sensitivity analysis to augment the experimental data.
    
    \begin{table}[b]
        \centering
        \captionsetup[table]{skip=3pt}
        \renewcommand{\arraystretch}{1.1} % Increase row spacing
        \setlength{\tabcolsep}{4.5pt} % Adjust column spacing
        \captionof{table}{\textbf{Parameter sets} and $R_0$ used in the three experiments: 
         $c_\phi$ is the proportion of admitted patients with resistance profile $\phi$, 
         $\tau$ denotes the probability that a patient is replaced with a new patient sampled from the community and 
         $\beta$ denotes the infection probability.}
    
    
        \label{tab:exp_par}
        \begin{tabular}{p{1.5cm}|ccccccc|c}
            \toprule
             \textbf{scenario} & $c_{S}$ & $c_{Ar}$ & $c_{Br}$ & $c_{ABr}$  & $c_{U}$ & $\tau$ & $\beta$ & $R_0$\\ 
             \midrule
             \textit{prevention}   & 0.75 & 0.05 & 0.05 & 0 & 0.15 & 0.20 & 0.30 & 1.5\\
             \textit{containment}  & 0.58 & 0.11 & 0.11 & 0.05 & 0.15 & 0.20 & 0.35 & 1.75\\
             \textit{maximum-emergence} & 0 & 0.50 & 0.50 & 0 & 0 & 0.50 & 0.25 & 0.5 \\
             \bottomrule
        \end{tabular}
    \end{table}
    
    
    %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
    \section*{Results}
    In each of our three experiments, we simulated the transition dynamics across six hospital wards on six 384-well plates. 
    Each 384-well plate simulates four replicate hospital wards, with each replicate comprising 96 wells representing 94 patients and two negative controls.
    We replace each assay plate daily to renew the treatment and medium (\sifig{1}). 
    Based on the turnover probability $\tau$, we randomly decide if a patient stays. If this is the case, we inoculate the well on the new plate from the same well on the old plate. 
    Else we replace this patient with a new incoming patient by inoculating the well on the new plate from a strain plate containing all resistance profiles.
    The resistance profile of the incoming patient is randomly selected based on predefined probabilities (sampling proportions $c_\phi$).
    Based on the infection probability $\beta$, we randomly decide if a patient will infect another randomly chosen patient. 
    These infections are then simulated \textit{in vitro} by passing a drop to the infected well on the new plate.
    All inoculations are carried out using the same pintool.
    
    
    \begin{figure}[h]
         \centering
         \includegraphics[width=\linewidth]{end_anova.pdf}
         \captionsetup{skip=2pt}
         \caption{
              Panels A--C show the mean frequency of uninfected (grey), single-resistant infected (blue), and double-resistant infected wells (green) during the last four transfers of the three scenarios.
              Circles represent replicates (n = 4), and bars represent means.
              Within resistance categories, bars not sharing a letter are significantly different (pairwise Tukey post hoc test, $p<0.05$; ANOVA tables and all p-values can be found in \sitab{34} -- \siref{50}).
          }
          \label{fig:end_anova}
    \end{figure}
    
    
    \paragraph{Multidrug strategies keep the overall number of infections lowest and best suppress single resistance.}
    The \textit{prevention}~scenario is characterised by a moderate proportion of single-resistant admissions to the hospital ward, the absence of pre-existing double resistance, and a moderately spreading infection dynamic ($R_0 = 1.5$, \sieq{1},  \sisec{SI Methods}).
    In this scenario, there are no differences between combination, mixing, and cycling on the frequency of uninfected, single-resistant-infected and double-resistant-infected wells (\panelref{fig:end_anova}{A}, time series in \sifig{2}).
    
    However, all multidrug strategies are significantly better at suppressing single resistance and increasing the number of cleared wells than the single-drug strategies and the control without treatment (\panelref{fig:end_anova}{A}). 
    In all scenarios, combination therapy was one of the most successful treatment strategies in minimising single-resistant and overall infections. 
    At the same time, we observed most single and double resistance in the untreated control.
    All strategies (but not the control) were able to clear sensitive infections effectively with clearance probabilities of 97\% for drug A, 73\% for drug B, and 86\% for AB (\sitab{8}).
    
    
    \begin{figure}
         \centering
         \includegraphics[width=\linewidth]{scen3_paper.pdf}
         \captionsetup{skip=2pt}
         \caption{
         Frequencies of resistance profiles over time during the \textit{maximum-emergence}~scenario. Panels (A--F) show the six tested strategies.
         Dots and bars show the mean and min/max interval of the four replicates.
         The dash-dotted line shows the mean value of 100 stochastic simulations based on the instruction set used in the \textit{in vitro} experiment (\textit{validation simulation}). 
         The solid line shows the mean value of 100 simulations with randomly created instruction sets based on the parameter set used in the experiment (\textit{variation simulations}). 
         The shaded error band indicates the 95-percentile interval between the \textit{variation simulations}.}
         \label{fig:exp3} 
    \end{figure}
    
    \paragraph{All treatment strategies fail to contain pre-existing double resistance.}
    The \textit{containment}~scenario explores a situation in which patients infected with double-resistant bacteria are continuously admitted to the hospital. 
    No strategy was able to contain the spread of double resistance, resulting in increased frequencies of double resistance ($> 40\%$) in all treatment arms at the conclusion of the experiment (\panelref{fig:end_anova}{B}).
    
    
    %% emergence of double resistance in scenario III
    \paragraph{Treatment strategies affect the emergence of double resistance.}
    In our experiments, double resistance primarily emerges in wells inoculated with both single-resistance plasmids via HGT, as the evolution of de novo resistance (e.g. by point mutations) to high drug concentrations ($>50 \times$MIC) is unlikely.
    As the inoculum volumes for turnover, infection, and passage are identical in our experiments, we do not distinguish between wells containing A-resistance ($A_r$) infecting wells containing B-resistance ($B_r$) or vice versa and simply refer to these events as superinfections.
    
    During the \textit{prevention} and \textit{containment} scenario, we could not identify differences in the strategies' abilities to suppress the emergence of double resistance. 
    We attribute this to a lack of statistical power because we observed only a few instances of double resistance emerging, mostly in the untreated control. 
    To address this, we selected parameters for the \textit{maximum-emergence}~scenario designed to maximise superinfection opportunities between wells carrying complementary resistance. 
    To this end, all admitted patients carried bacteria with only one of the two resistance plasmids (at equal proportions). 
    In addition, we set the probability of admission and discharge to $\tau = 0.5$ and the infection probability to $\beta = 0.25$, resulting in a basic reproduction number $R_0 = 0.5$ (\sieq{1}).
    An $R_0<1$ makes double resistance more likely to be replaced by newly admitted patients than to spread, thus maintaining a high potential for emergence.
    We implement this scenario to explore emergence under a magnifying glass, being aware that it does not reflect a likely clinical situation. In this scenario, combination therapy and monotherapy with drug A lead to the lowest frequency of double resistance during the last four transfers (\panelref{fig:end_anova}{C}, \autoref{fig:exp3}).
    
    For the \textit{maximum-emergence} scenario, we observed that combination therapy, cycling, and monotherapy with drug A were most effective in preventing newly emerging double resistance. 
    Combination was the only strategy in which we did not observe a single case of emerging double resistance after the first transfer (\panelref{fig:emergence}{A}).
    Furthermore, combination therapy is the most successful treatment strategy in minimising the number of both single-resistant and overall infections, while the control leads to the highest number of double- and single-resistant, and overall infections.
    
    \paragraph{Combination therapy suppresses the emergence of double resistance by preventing superinfections.}
    Treatment strategies can impact the emergence of double resistance by suppressing superinfections.
    The number of superinfections $n_\mathcal{S}$ is dependent on the abundance of the single resistance carrying wells $A_r$ and $B_r$.
    Hence, we expect the highest number of superinfections and most opportunities for emerging double resistance when both single resistances are unaffected by the treatment and the fewest if the treatment successfully suppresses both single resistances. 
    Our measurements confirmed these expectations during the \textit{maximum-emergence}~scenario.
    Here $n_\mathcal{S}$ is highest in the control group (no treatment) and lowest under combination therapy (\panelref{fig:emergence}{B}).
    
    \paragraph{Treatment strategies influence the emergence of double resistance within superinfected wells.}
    We observed the highest average frequency of superinfections developing double resistance $(\frac{n_\mathcal{E}}{n_\mathcal{S}})$ in antibiotic-free medium and in medium treated with antibiotic B (tetracycline). 
    In contrast, superinfections resulting in double resistance rarely occur in medium treated with antibiotic A (ceftazidime) or both drugs (\panelref{fig:emergence}{C}).
    We think the impact of treatment on cell densities within superinfected wells (both in infected and infecting wells) can best explain these findings.
    
    Firstly, applying a drug affects the in-well population dynamics of superinfected wells.
    Reducing the cell density for one or both single-resistant strains within a superinfected well reduces the probability of bacteria with complementary resistance to encounter and conjugate (see \sisec{SI Results}). 
    As drug A (bactericidal) decreases the cell density faster than drug B (bacteriostatic), more conjugation opportunities occur in wells treated with drug B.
    
    Secondly, the treatment strategies influence the number of transferred single-resistant bacteria that inoculate superinfections by curbing the bacterial density within the infecting wells (see \sisec{SI Results}).
    
    Due to the differences in the abilities of drugs A and B to prevent conjugation, there are times (cycles) and places (beds) during cycling and mixing where using drug B offers increased opportunities for the emergence of double resistance, which is never the case with combination therapy.
    
    
    \begin{figure*}[t]
        \newcommand{\imageheight}{2.95cm}
    %    \subpanel[xshift = 0pt, yshift=10pt]{.33\linewidth}{
                \includegraphics[height = \imageheight]{20220412_f_emergence_anova.pdf}
            \includegraphics[height = \imageheight]{20220412_encounter_frequencies.pdf}
            \includegraphics[height = \imageheight]{20220412_conjugation_prop.pdf}
         
        \captionsetup{skip=2pt}
       \caption{
            Analysis of the emergence of double resistance \textit{in vitro} and superinfection between single resistant $A_r$ and $B_r$ wells during the \textit{maximum-emergence}~scenario, from transfer four onwards. 
            Each dot corresponds to data from a single plate, with each plate representing a distinct treatment arm, encompassing 376 patients for one transfer.
            Mean values are represented by red bars. 
            Bars not sharing a letter are significantly different ($p < 0.05$, ANOVA tables and pairwise Tukey post hoc results can be found in \sitab{51} -- \siref{56}).  
            \textbf{A)} number of newly emerged cases of double resistance per plate ($n_{\mathcal{E}}$), normalised to the total number of patients (wells) per plate ($n_P = 376$). 
            Here the stars indicate predictions based on the ratio of superinfections per patient and the emergence per superinfection (\sisec{SI Computational Model}).
            \textbf{B)} Number of superinfections per plate ($n_\mathcal{S}$), normalised to the total number of patients (wells) per plate  ($n_P = 376$). 
            \textbf{C)} Number of superinfected wells treated with  (A, B, AB and none) that develop double resistance ($n_\mathcal{E}$) divided by the number of all superinfected wells ($n_\mathcal{S}$) receiving the respective treatment.
        }
        \label{fig:emergence}
    \end{figure*}
    
    \paragraph{Computational model corroborates the robustness of experimental outcomes.}
    The experiments are conducted by a liquid handling platform that carries out predefined instructions, specifying which infections occur and who is admitted or discharged. 
    The instructions are randomized based on parameter sets we defined for each scenario, including the overall infection and turnover probability as well as the distribution of the resistance profiles of admitted patients.
    We call the entirety of all instructions that come up during one experiment an `instruction set'.
    Due to the scale and technical complexity of the experiments, it was not feasible to carry out individual instruction sets for each replicate, so we opted to apply the same instruction sets for all replicates.
    This raises the question of whether the experimental results are a consequence of a specific instance of this random process and whether they are robust to the randomisation in the instruction set.
    To address this, we developed a discrete-time stochastic model comprising 94 individual \textit{in silico} patients mimicking the epidemiological dynamics of the experiment (\sisec{SI Computational Model}). 
    The model was parametrised, but not fitted, with transition probabilities (\sitab{18}--\siref{25}) that we estimated based on the transition frequencies measured \textit{in vitro}.
    We used the same transition probabilities in the simulations for all scenarios.
    
    First, we validated the model by averaging 100 \textit{validation simulations}, each employing the identical instruction sets used \textit{in vitro}.
    The aim of the \textit{validation simulations} is to recreate the experiments \textit{in silico} (\sifig{3B}).
    We found that the simulation results are in good agreement with the experimental data, indicating that the model reflects the dynamics observed in the \textit{in vitro} experiments well (see \autoref{fig:exp3}, \sifig{2}, and \sifig{4}).
    One exception is the spread of A-resistance during the \textit{prevention}~scenario in control and Mono~A.
    This could indicate an increased number of contaminations at the beginning of the \textit{prevention}~scenario.
    We also observe some discrepancies for the spread of double resistance during the \textit{prevention}~scenario, which we attribute to contamination artefacts in the transition probabilities (see \sisec{SI Computational Model}).
    
    Second, we averaged 100 \textit{variation simulations} to assess the robustness of the experimental outcomes against variations in the instruction sets. 
    In these \textit{variation simulations}, each of the 100 instruction sets was randomized based on the same three parameter sets used \textit{in vitro} (\sifig{3C}). 
    Differences between the \textit{validation} and \textit{variation simulations} indicate differences in outcome due to the randomization of the instruction sets.
    For instance, with a turnover probability $\tau = 0.2$ and an admission probability $c_A = 0.05$, we expect $0.94$ $A_r$ admissions per transfer. However, random fluctuations can result in either more (or fewer) $A_r$ admissions, leading to a temporarily higher (or lower) frequency of $A_r$ in the \textit{validation simulations}, creating a temporary spread between the \textit{validation} and \textit{variation simulations}.
    We observed that the \textit{validation simulations} fluctuate around the \textit{variation simulations} and never diverge far (see \autoref{fig:exp3}, \sifig{2}, and \sifig{4}), indicating robustness of the experimental results to the randomisation of the instruction sets.
    
    
    \paragraph{\textit{In silico} sensitivity analysis indicates that the superiority of combination therapy is robust.}
    Given that the \textit{validation simulations} agreed well with the experiments, we used the model to perform an {\it in silico} parameter sensitivity analysis of the experimental results (\sifig{3D}). 
    To this end, we ran ten simulations for each of 20,000 randomly generated parameter sets by varying the turnover and infection probability and the five sampling proportions for incoming patients: ($\tau$, $\beta$, $c_S$, $c_{A_r}$, $c_{B_r}$, $c_{AB_r}$, $c_U$).
    For half of the parameter sets, we forced the frequency of incoming patients with double resistance ($c_{AB_r}$) to zero. 
    
    We used the frequency of uninfected \textit{in silico} patients to measure treatment success. 
    Using this criterion, the control strategy (no treatment) always performed worst, and accordingly, we excluded this treatment arm from the following analysis.
    Strategies were then classified as (i) 'single winners' if they are significantly better than all other strategies; (ii) 'winners' if they are not outperformed by any other strategy; (iii) 'losers' if they do not outperform at least one other strategy; or (iv) 'single losers' if all other strategies outperform them.
    
    In parameter sets with and without pre-existing double resistance, combination therapy ranks most often as one of the best strategies (87\%  and 98\%, respectively). 
    It is the single best strategy in 55\% of the tested parameter sets with pre-existing double resistance and in 93\% of cases without pre-existing double resistance (\autoref{fig:senstivity_analysis}, \sitab{14},~\sitab{15}). 
    
    In some situations (for example, when one strategy is much worse than all others), it is more important to avoid the worst strategy than selecting the very best strategy among the good ones. 
    Our analysis finds that combination therapy is almost never among the worst strategies, while usually one of the two monotherapies performs worst.
    As expected, single-drug strategies perform particularly poorly when there is a high frequency of pre-existing single-resistance to the applied drug (\sitab{11}, \sitab{13}).
    
    Cycling and mixing lose substantially less than the monotherapies but are rarely the single best strategy. 
    
    \begin{figure}[h]
        \centering
        \includegraphics[width = \linewidth]{wins_and_losses_preexisting.pdf}
        \includegraphics[width = \linewidth]{wins_and_losses_no_preex.pdf}
        \captionsetup{skip=0pt}
          \caption{
                Effectiveness of the five treatment strategies in maximising the frequency of uninfected individuals across randomly generated parameter sets.
                Strategies not significantly better than any other are marked as losers (pastel red), and those significantly worse than all others as single losers (dark red). 
                Strategies not significantly worse than any other are classified as winners (pastel blue), and those significantly better than all others as single winners (dark blue). 
                Strategies without significant differences were excluded.
                \textbf{(A)} 10,000 parameter sets with pre-existing double resistance. 606/10,000 sets yielded no significant difference between the strategies.
                \textbf{(B)} 10,000 parameter sets without pre-existing double resistance. 100/10,000 sets yielded no significant difference between the strategies.
                }
        \label{fig:senstivity_analysis}
    \end{figure}
    
    %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
    \section*{Discussion}
    In our study, multi-drug strategies, particularly combination therapy, outperformed monotherapies in reducing overall infections and the emergence of double resistance across most scenarios, while we observed most emergence of double resistance in the untreated control.
    Interestingly, the effectiveness of combination therapy does not stem from an increased efficacy associated with higher doses. 
    This is because an asymmetrical antagonism exists between the bactericidal antibiotic ceftazidime (drug A) and the bacteriostatic antibiotic tetracycline (drug B), resulting in a lower clearance rate for the combination A+B compared to drug A alone (\sisec{SI Results}). 
    This observation implies that combination therapy may be even more advantageous when drugs are neutral or synergistic towards each other.
    
    Why does the absence of treatment lead to worse outcomes, and why is combination therapy preventing the emergence of double resistance so effectively?
    
    First, we measured the presence, not the density, of resistant bacteria in wells by assessing if small aliquots of the liquid culture could grow on treated agar plates. 
    This approach quantifies the number of wells hosting a specific resistance but can not quantify the frequency of resistance in the in-well population. 
    The information about presence/absence alone yields important information about potential treatment success and is used in analogous clinical diagnostic methods, such as disk diffusion tests \cite{eucast_disk2024}.
    
    We would only recognize a loss of resistance (in the experiments and clinical samples) if the resistant strain were fully outcompeted. 
    This was not observed during the \textit{containment} scenario in the untreated control.
    Such an outcome was expected due to the short average patient stay of 2--5 days in our experiments and 5--6 days in clinical situations \cite{BAG2015}.
    For the same reason, we would not expect an eradication of resistance but only a shift in resistance density, even if there were more substantial costs of resistance or higher segregational loss.
    In our experiments, we found no evidence of a cost of resistance (see \sisec{SI Methods}, \sifig{6}, and \sitab{3}) or segregational loss (see \sisec{SI Methods} and \sitab{4}).
    
    Second, in our experiments, the emergence of double resistance requires conjugation, which relies on superinfection between hosts with complementary resistance profiles.
    As demonstrated in \panelref{fig:emergence}{B}, the lowest number of superinfections occur in combination therapy, where both single-resistant strains can be cleared. 
    Conversely, without treatment, the abundance of single resistance is highest resulting in the highest number of superinfections.
    
    Third, the applied antibiotics affect the frequency of superinfections leading to double resistance, likely by influencing the growth dynamics within the superinfected well and potentially the conjugation rate \cite{Headd2018}. 
    However, our experimental data are unsuitable for supporting or rejecting the impact on conjugation rates.
    We observed the least emergence of double resistance in superinfected wells treated with both drugs and most in untreated wells, contributing to the superiority of combination therapy and the high rates of double resistance in the absence of treatment (\panelref{fig:emergence}{C}).
    This effect on the in-well dynamics may be a property of the chosen drugs and concentrations, and we expect better results for cycling and mixing if both drugs were equally effective in suppressing double resistance or worse results for combination therapy if the combination of both drugs was less effective.
    
    Fourth, we observed that the number of single-resistant bacteria inoculating superinfections impacts the frequency of emerging double resistance (see \sisec{SI Results}, \sitab{1}). 
    In our setup, superinfected wells receive two inocula, with at least one inoculum transferred from the previous plate (by infection) that has already undergone treatment for one day. 
    When prior treatment led to a low bacterial density in the source wells, we did not observe any cases of double resistance emerging.
    This could magnify the effectiveness of combination therapy, where all potential single-resistant inocula transferred from the previous plate contain low bacterial densities due to effective treatment. 
    On the one hand, this may be more a characteristic of our experimental setup due to the fixed length of the treatment interval and high clearance probabilities. 
    On the other hand, we indeed expect fewer cases of emergence in superinfected patients if the infecting inocula are small.
    
    In our experiments and simulations, combination therapy showed superior results in minimizing infections and preventing double resistance. 
    This advantage may partly result from assumptions and simplifications, including the chosen strain, drugs, plasmids, and inoculum size, the discrete setup with fixed treatment durations, colonization-independent infection and turnover probabilities, and the absence of an immune system and microbiome. 
    Also, treating all patients irrespective of colonization diverges from clinical reality in two ways: 
    i) in a clinical setting, some untreated patients may serve as a sanctuary for resistance and a potential source of double resistance and 
    ii) treating all patients, regardless of infection status, contrasts with clinical efforts to promote targeted antibiotic use. 
    However, since patients as we model them in our \textit{in vitro} experiments lack a microbiome, treating uninfecteds should have no impact on the resistance dynamics.
    
    Despite the numerous differences between our experiments and a real clinical situation, we argue that the relative effectiveness of combination therapy in suppressing double resistance would likely translate to real patients.
    The reason is that the emergence of double resistance hinges on two critical processes: 1) preventing superinfections between patients carrying bacteria with complementary resistance plasmids and 2) the probability that superinfected hosts develop double resistance.
    We think that combination therapy offers a strategic advantage in addressing both processes.
    
    Our results complement the findings by Angst et al. \cite{Angst2021}, who observed similar outcomes in the context of chromosomal resistance. We believe that such \textit{in vitro} experimental models, which explore admittedly idealised and simplified epidemiological scenarios, can help to bridge the divide between mathematical models and randomised clinical trials. However, ultimately the evidence for or against the benefits of combination therapy must be confirmed by rigorous clinical trials with sufficient statistical power to support or challenge the effectiveness of combination therapy.
    
    
    
   
   \showmatmethods{} % Display the Materials and Methods section

    \showacknow{} % Display the acknowledgments section
    
    \bibsplit[9]
    %Use \bibsplit to split the references from the body of the text. Value "[2]" represents the number of reference in the left column (Note: Please avoid single column figures & tables on this page.)
    %TC:ignore
    % Bibliography
    \bibliography{TSPlasmid}
%    \printbibliography
    %TC:endignore
\end{document}